wlauto/utils/fps.py

```diff
@@ -19,8 +19,6 @@ class FpsProcessor(object):
     Provide common object for processing surfaceFlinger output for frame
     statistics.
 
-    This processor adds four metrics to the results:
```

Comment:
Originally the code added the metrics to the context but that has now changed. This comment is therefore a bit misleading.